src/index.html

```diff
@@ -1,12 +1,57 @@
 <!DOCTYPE html>
-<html lang="en">
+<html
+  class="page"
+  lang="en"
+>
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <main class="page__main">
+      <section
+        class="calendar calendar--start-day--fri calendar--month-length--28 page__calendar"
+      >
```

Comment:
Use div instead of section
```suggestion
      <div
        class="calendar calendar--start-day--fri calendar--month-length--28 page__calendar"
      >
```